docs/Developers/Git_Basics.rst

```diff
@@ -453,3 +453,9 @@ use an interactive rebasing session. This opens an editor where you can enter
 commands (described below) for each commit to be rebased. These commands
 determine how individual commits will be transferred to the new base. You can
 also reorder the commit listing to change the order of the commits themselves.
+
+If you would like more information/commands, please use your favourite search
+engine to look for it. You could also read the
+`git documentation <https://git-scm.com/doc>`_ to learn more. Git is widely used
```

Comment:
I'd also vote for leaving the link in as it's a good starting point to get to know git basics and does not eliminate the need to look for things entirely.